npc/re/merchants/ammo_boxes.txt

```diff
@@ -262,7 +262,7 @@ S_Slug_Exchange:
 				case 1:
 					mes "[Kenny]";
 					mes "All done.";
-					delitem .@slug_id, .@new_count * 500;
+					delitem .@slug_id, .@new_count * .@req_count;
```

Comment:
Should also be done above, where it is output via "mes" or not?
There `.@count` is used directly.

Reply:
if do that, output via "mes" is use .@count countitem, which doesn't match char inventory

what should I do? please advise

Reply:
About that, I don't know how it is on official, I'm in favor of leaving it like that for now